lib/node_modules/@stdlib/ndarray/base/assert/is-single-segment-compatible/src/main.c

```diff
@@ -49,7 +49,7 @@
 * int8_t b = stdlib_ndarray_is_single_segment_compatible( STDLIB_NDARRAY_UINT8, ndims, shape, strides, offset );
 * // returns 1
 */
-int8_t stdlib_ndarray_is_single_segment_compatible( enum STDLIB_NDARRAY_DTYPE dtype, int64_t ndims, int64_t *shape, int64_t *strides, int64_t offset ) {
+int8_t stdlib_ndarray_is_single_segment_compatible( enum STDLIB_NDARRAY_DTYPE dtype, int64_t ndims, int64_t *shape, const int64_t *strides, int64_t offset ) {
```

Comment:
@GeoDaoyu In general, for changes such as this, you also need to update the header file and the README (if the function signature is documented there).